### PR TITLE
LPS-137424 Added else if for actionId parameter for complex SQL query

### PR DESCRIPTION
**Pull request by mbowerman**
From @javalosjr96

> Description:
> If the user adds:
> `groups.complex.sql.class.names=com.liferay.portal.kernel.model.User,com.liferay.portal.kernel.model.Organization,com.liferay.portal.kernel.model.UserGroup
> `
> to the portal-ext.properties, multiple exceptions are thrown in the console when the User tries to access their site membership.
> This is due to the parameter key "actionId" not being handled and is being used in the complex SQL query.
> ```
> Positional parameter does not exist: 11 in query:   SELECT DISTINCT Group_.groupId AS groupId FROM Group_  WHERE  (Group_.liveGroupId = 0) AND (Group_.active_ = ?)    AND  Group_.manualMembership = ?    AND  Group_.site = ?    AND ((Group_.classNameId = ?) OR (Group_.classNameId = ?)) AND  (Group_.companyId = ?) AND (Group_.parentGroupId != ?) AND (Group_.liveGroupId = 0) AND (Group_.groupKey != 'Control Panel') AND (LOWER(Group_.name) LIKE ? OR ? IS NULL) AND (LOWER(Group_.description) LIKE ? OR ? IS NULL) AND (Group_.type_ != 4)
> javax.portlet.PortletException: org.apache.jasper.JasperException: com.liferay.portal.kernel.exception.SystemException: java.lang.IllegalArgumentException: Positional parameter does not exist: 11 in query:   SELECT DISTINCT Group_.groupId AS groupId FROM Group_  WHERE  (Group_.liveGroupId = 0) AND (Group_.active_ = ?)    AND  Group_.manualMembership = ?    AND  Group_.site = ?    AND ((Group_.classNameId = ?) OR (Group_.classNameId = ?)) AND  (Group_.companyId = ?) AND (Group_.parentGroupId != ?) AND (Group_.liveGroupId = 0) AND (Group_.groupKey != 'Control Panel') AND (LOWER(Group_.name) LIKE ? OR ? IS NULL) AND (LOWER(Group_.description) LIKE ? OR ? IS NULL) AND (Group_.type_ != 4)
> ```
> 
> 
> **Steps to reproduce**
> 
> - Add the following property to the portal-ext.properties file in a clean bundle of Liferay DXP 7.2 installed with dxp-14:
> `groups.complex.sql.class.names=com.liferay.portal.kernel.model.User,com.liferay.portal.kernel.model.Organization,com.liferay.portal.kernel.model.UserGroup`
> - Start the bundle and log in as the Test Test user.
> - Go to Control Panel -> Users -> Users and Organizations and select the Test Test user
> - Select the Membership tab and then select the Select button next to the sites section.
> 
> **Solution**
> 
> - Added else if for "actionId" parameter.

https://issues.liferay.com/browse/LPS-137424

/cc @javalosjr96

### CODE REVIEW

**Comment by mbowerman** [comment]
ci:test:sf

**Comment by mbowerman** [comment]
ci:test:relevant

**Comment by Sharryxue** [comment]
The failed tests in ci:test:relevant test suite not related.

**Comment by javalosjr96** [comment]
Nice

File: portal-impl/src/com/liferay/portal/service/persistence/impl/GroupFinderImpl.java [hunk 1]
```diff
@@ -1230,6 +1230,8 @@ else if (key.equals("groupsTree")) {
 					}
 				}
 			}
+			else if (key.equals("actiondId")) {
+			}
```
Comment (drewbrokke) [inline portal-impl/src/com/liferay/portal/service/persistence/impl/GroupFinderImpl.java:1234]: Is this a typo? I imagine it should be `"actionId"`.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@mbowerman @javalosjr96 this looks fine to me, but I think there's a typo in the string. 

Could you please add an integration test case to assert that "actionId" is used during index search and ignored during database search?

Once you do, please send to the @liferay-echo user and mention me on the pull. Thank you!

**Pull request closed without merging**